klayout/drc/run_drc.py

```diff
@@ -0,0 +1,709 @@
+# Copyright 2022 GlobalFoundries PDK Authors
```

Comment:
looks like flake8 is ignoring most of the violations:
```
ignore = E203, E266, E501, W503, F403, F401
```
when we turn those on we get:
```
./klayout/drc/run_drc.py:20:89: E501 line too long (329 > 88 characters)
./klayout/drc/run_drc.py:25:89: E501 line too long (132 > 88 characters)
./klayout/drc/run_drc.py:26:89: E501 line too long (103 > 88 characters)
./klayout/drc/run_drc.py:27:89: E501 line too long (103 > 88 characters)
./klayout/drc/run_drc.py:28:89: E501 line too long (103 > 88 characters)
./klayout/drc/run_drc.py:31:89: E501 line too long (112 > 88 characters)
./klayout/drc/run_drc.py:32:89: E501 line too long (92 > 88 characters)
./klayout/drc/run_drc.py:34:89: E501 line too long (112 > 88 characters)
./klayout/drc/run_drc.py:87:89: E501 line too long (117 > 88 characters)
./klayout/drc/run_drc.py:132:89: E501 line too long (171 > 88 characters)
./klayout/drc/run_drc.py:138:89: E501 line too long (147 > 88 characters)
./klayout/drc/run_drc.py:145:89: E501 line too long (93 > 88 characters)
./klayout/drc/run_drc.py:150:89: E501 line too long (106 > 88 characters)
./klayout/drc/run_drc.py:152:89: E501 line too long (92 > 88 characters)
./klayout/drc/run_drc.py:156:89: E501 line too long (106 > 88 characters)
./klayout/drc/run_drc.py:168:13: W503 line break before binary operator
./klayout/drc/run_drc.py:169:13: W503 line break before binary operator
./klayout/drc/run_drc.py:170:13: W503 line break before binary operator
./klayout/drc/run_drc.py:233:9: W503 line break before binary operator
./klayout/drc/run_drc.py:234:9: W503 line break before binary operator
./klayout/drc/run_drc.py:235:9: W503 line break before binary operator
./klayout/drc/run_drc.py:241:89: E501 line too long (120 > 88 characters)
./klayout/drc/run_drc.py:264:89: E501 line too long (101 > 88 characters)
./klayout/drc/run_drc.py:275:89: E501 line too long (95 > 88 characters)
./klayout/drc/run_drc.py:280:89: E501 line too long (113 > 88 characters)
./klayout/drc/run_drc.py:305:89: E501 line too long (90 > 88 characters)
./klayout/drc/run_drc.py:310:89: E501 line too long (90 > 88 characters)
./klayout/drc/run_drc.py:315:89: E501 line too long (90 > 88 characters)
./klayout/drc/run_drc.py:358:89: E501 line too long (91 > 88 characters)
./klayout/drc/run_drc.py:380:89: E501 line too long (105 > 88 characters)
./klayout/drc/run_drc.py:384:34: E203 whitespace before ':'
./klayout/drc/run_drc.py:387:89: E501 line too long (105 > 88 characters)
./klayout/drc/run_drc.py:394:89: E501 line too long (127 > 88 characters)
./klayout/drc/run_drc.py:412:89: E501 line too long (90 > 88 characters)
./klayout/drc/run_drc.py:447:89: E501 line too long (89 > 88 characters)
./klayout/drc/run_drc.py:456:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:509:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:514:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:519:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:519:89: E501 line too long (99 > 88 characters)
./klayout/drc/run_drc.py:525:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:530:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:554:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:584:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:595:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:606:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:614:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:619:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:634:89: E501 line too long (113 > 88 characters)
./klayout/drc/run_drc.py:646:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:649:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:654:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:658:5: E266 too many leading '#' for block comment
./klayout/drc/run_drc.py:663:9: W503 line break before binary operator
./klayout/drc/run_drc.py:664:9: W503 line break before binary operator
./klayout/drc/run_drc.py:689:9: W503 line break before binary operator
./klayout/drc/run_drc.py:690:9: W503 line break before binary operator
```
I think we may want to turn those on? wdyt?

Reply:
As agreed in another conversation, we will target this in another PR. Please find Issue filed for this:
https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/issues/4